lm_eval/tasks/benchmarks/multimedqa/multimedqa.yaml

```diff
@@ -5,19 +5,13 @@ task:
   - medqa_4options
   - task: mmlu_anatomy
     task_alias: "anatomy (mmlu)"
-    group_alias: null
```

Comment:
Is this change confirmed to not disturb multimedqa printing?

Reply:
Yep, it's been confirmed to work.